types/Cargo.toml

```diff
@@ -41,8 +41,10 @@ bincode = "1.3.1"
 criterion = "0.3.5"
 getrandom = { version = "0.2.0", features = ["rdrand"] }
 proptest = "1.0.0"
+proptest-attr-macro = "1.0.0"
 serde_json = "1.0.55"
 serde_test = "1.0.117"
+serde_cbor = "0.11.1"
```

Comment:
```suggestion
```

Reply:
Fixed in [e93dc37](https://github.com/nilz3ro/casper-node/pull/5/commits/e93dc3787a9983c352ebfdba0fc6bccb8f4d7c41).